live-css-editor/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import './App.css';
+import Input from './input'
+
+class App extends Component {
+  state = {
+    style: ''
+  }
+
+  changeStyle = style => {
+    this.setState({style})
+  }
+  render() {
+    
+    return (
+      <div className="App">
+        <h1>Live CSS Editor</h1>
+        <Input 
+          changeStyle={this.changeStyle}
+        />
+        <style>{this.state.style}</style>
+      </div>
```

Comment:
The page has no other content but the title. It would've been nice to have more elements to play with and test the editor.